### ISSUE TITLE
a11y: Color Contrast - Cluster Drivers

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13811]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3  
Ratio: 3.16:1
Main Content
- Refresh Kubernetes Metadata
- Create
- Deactivate Button

Foreground Text Color: #FFFFFF
Background Text Color: #297DB4
Ratio: 4.48:1
Main Content
- Refresh Kubernetes Metadata, Create button and - Deactivate Button (on focus)

Foreground Text Color: #5D995D
Background Text Color: #FFFFFF
Ratio: 3.39:1
Main Content 
- Active Button 

Foreground Text Color: #5D995D
Background Text Color: #E9EEF1
Ratio: 2.90:1
Main Content 
- Active Button 

Foreground Text Color: #FFFFFF
Background Text Color: #F64747
Ratio: 3.55:1
Main Content 
- Inactive Button 

Foreground Text Color: #B6B6C2
Background Text Color: #FFFFFF
Ratio: 2.00:1
Main Content 
- https://drivers.rancher.cn/kontainer-engine-driver-aliyun/0.2.6/kontainer-engine-driver-aliyun-linux, Built In, 
 https://drivers.rancher.cn/kontainer-engine-driver-baidu/0.2.0/kontainer-engine-driver-baidu-linux and so on 
       

Foreground Text Color: #B6B6C2
Background Text Color: #E9EEF1
Ratio: 1.71:1 
Main Content 
- https://drivers.rancher.cn/kontainer-engine-driver-aliyun/0.2.6/kontainer-engine-driver-aliyun-linux, Built In, 
 https://drivers.rancher.cn/kontainer-engine-driver-baidu/0.2.0/kontainer-engine-driver-baidu-linux and so on (on hover) 
      
## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Issue 176 resolved.
